docs/pages/access-controls/access-graph/self-hosted.mdx

```diff
@@ -29,8 +29,19 @@ to Teleport Enterprise customers.
 - A TLS certificate for the Access Graph service
   - The TLS certificate must be issued for "server authentication" key usage,
     and must list the IP or DNS name of the TAG service in an X.509 v3 `subjectAltName` extension.
+  - Starting from version 1.20.4 of the Access Graph service, the container by default runs as a non-root user.
```

Comment:
nit: "the container runs as a non-root user by default." might be a nicer phrasing?